tensorlayer/layers/convolution.py

```diff
@@ -1560,79 +1549,87 @@ def deconv2d(prev_layer,
     b_init : initializer or None
         The initializer for the bias vector. If None, skip biases.
     W_init_args : dictionary
-        The arguments for the weight matrix initializer.
+        The arguments for the weight matrix initializer (For TF < 1.3).
     b_init_args : dictionary
-        The arguments for the bias vector initializer.
+        The arguments for the bias vector initializer (For TF < 1.3).
     name : str
         A unique layer name.
 
-    Returns
-    -------
-    :class:`Layer`
-        A :class:`DeConv2dLayer` object.
-
     """
 
```

Comment:
Please re-add the deprecation header, just above the init function

Reply:
done